runbotics-orchestrator/src/main/java/com/runbotics/service/ProcessCollectionService.java

```diff
@@ -1,7 +1,12 @@
 package com.runbotics.service;
 
+import com.runbotics.service.criteria.ProcessCollectionCriteria;
 import com.runbotics.service.dto.ProcessCollectionDTO;
 
+import java.util.List;
+
 public interface ProcessCollectionService {
+
+    List<ProcessCollectionDTO> getCollectionsByParentId(ProcessCollectionCriteria criteria);
```

Comment:
if we pass criteria, than it's more by criteria than by id isn't it?
we may have more filters in the future